File: MapTest/results/MapTest.php
<?php

class TestClass {
    function getResult() {
        $map_obj = array(
          "x" => 5,
        );
        //let containsX = "x" in mapObj;
        //delete mapObj["x"];
        $map_obj["x"] = 3;
        return $map_obj["x"];
    }
    
    function testMethod() {
        print(("Result = " . ($this->getResult()) . "") . "\n");
    }
}

try {
    $c = new TestClass();
    $c->testMethod();
} catch (Exception $err) {
    echo 'Exception: ' . $err->getMessage() . "\n";
}

File: MapTest/results/MapTest.rb
class TestClass 
  def get_result()
      map_obj = {
        "x" => 5,
      }
      #let containsX = "x" in mapObj;
      #delete mapObj["x"];
      map_obj["x"] = 3
      return map_obj["x"]
  end

  def test_method()
      puts "Result = #{self.get_result()}"
  end
end

begin
    TestClass.new().test_method()
rescue Exception => err
    print "Exception: #{err}"
end